rasa/core/brokers/pika.py

```diff
@@ -431,17 +480,18 @@ def _get_message_properties(
     def _basic_publish(
         self, body: Text, headers: Optional[Dict[Text, Text]] = None
     ) -> None:
-        self.channel.basic_publish(
-            "",
-            self.queue,
-            body.encode(DEFAULT_ENCODING),
-            properties=self._get_message_properties(headers),
-        )
+        for queue in self.queues:
+            self.channel.basic_publish(
+                "",
+                queue,
+                body.encode(DEFAULT_ENCODING),
+                properties=self._get_message_properties(headers),
+            )
```

Comment:
Should this be a `fanout` exchange? 
https://www.cloudamqp.com/blog/2015-09-03-part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html#fanout-exchange
https://www.rabbitmq.com/tutorials/tutorial-three-python.html